stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -90,7 +91,10 @@ internal class PaymentOptionsAdapter(
         position: Int,
         isClick: Boolean
     ) {
-        if (position != NO_POSITION && (canClickSelectedItem || position != selectedItemPosition)) {
+        if (interactionEnabled &&
+            position != NO_POSITION &&
+            (canClickSelectedItem || position != selectedItemPosition)
+        ) {
```

Comment:
Can we disable the views instead?

Reply:
I tried and for that to work we'll need a small refactor. I'll do that as a follow-up which includes making them look disabled.